lib/liquidTestRunner.js

```diff
@@ -343,7 +346,8 @@ async function runTests(
   handle,
   testName = "",
   previewOnly = false,
-  renderMode = "none"
+  renderMode = "none",
+  skipSpinner = false
```

Comment:
Skipping the spinner is not really necessary. Is not the reason of this bug. 
But we could add it if you think it's useful.

Reply:
I'll remove it, cause it wouldn't have really solved the issue I wanted and would take a bigger code change. So would take it into separate PR